src/views/Error.vue

```diff
@@ -22,15 +25,24 @@
       <!-- failsafe begins -->
       <div v-if="hasFailSafe">
 
+        <!-- TODO - Adding loading spinner till YT video loads -->
```

Comment:
As the loading up of the Youtube embed takes some time (because the youtube id is being fetched from a get request), need to add a loading spinner here as well

Reply:
YouTube is passed as part of the value itself right? So, what is the fetch part here?

Reply:
Yes, nothing is being fetched here. Youtube ID comes from `player.vue` and it will already be present when this error page shows up. For Pritam's reference, I just used wrong terminology here. Nothing is being fetched.

Reply:
can remove this comment then right?

Reply:
yes, will do

Reply:
Haan nikal do